src/tcgwars/logic/impl/gen4/RisingRivals.groovy

```diff
@@ -659,22 +850,37 @@ public enum RisingRivals implements LogicCardInfo {
           pokeBody "Sticky Hold", {
             text "If Gastrodon East Sea is switched or retreats to your Bench, move as many Energy cards attached to Gastrodon East Sea as you like to the new Active Pokémon."
             delayedA {
+              after FALL_BACK, self, {
+                if(self.cards.energyCount(C) && confirm("Use Sticky Hold?")) {
+                  def energyList = self.cards.select(max:self.cards.filterByType(ENERGY).size(),"Move any number of Energy cards attached from $self to ${self.owner.pbg.active}",cardTypeFilter(ENERGY))
+                  energyList.each {
+                    energySwitch(self,self.owner.pbg.active,it)
+                  }
+                }
+              }
             }
           }
           move "Calling Wave", {
             text "Search your deck for up to 2 Gastrodon and put them onto your Bench as Basic Pokémon. Put 2 damage counters on each of them. Shuffle your deck afterward."
             energyCost C
-            attackRequirement {}
+            attackRequirement {
+              assert my.deck : "Your deck is empty"
+              assert my.bench.notFull : "Your bench is full"
+            }
             onAttack {
-              damage 0
+              def max = Math.min(2,my.bench.freeBenchCount)
+              my.deck.search(max:max,"Search your deck for up to 2 Gastrodon to put onto your bench",{it.cardTypes.is(POKEMON) && it.name.contains("Gastrodon")}).each {
+                def pcs = benchPCS(it)
+                directDamage 20, pcs, SRC_ABILITY
```

Comment:
Kinda unrelated
### 
Things like this makes thinking of how to implement auto source really annoying. PokemonPowers are not Abilities, so manually setting the source to Ability for them is kind of problematic. I suppose when we do do that we should just ignore all manual setting currently being done.